core/platform/taskqueue/gae_taskqueue_services.py

```diff
@@ -36,6 +38,18 @@ def defer_to_events_queue(fn, *args, **kwargs):
     # details on the _queue kwarg.
     deferred.defer(fn, *args, _queue=QUEUE_NAME_EVENTS, **kwargs)
 
+def enqueue_task(url, params, countdown):
+    """Adds a new task for sending email.
+
+    - params sepcifies parameters that will be passed as payload to handler
```

Comment:
typo: specifies

Though here's a suggested formatting (based on the google python style guide):

```
Args:
- url: ...
- params: the parameters that will be passed as a payload to the
      handler function
- countdown: the amount of time, in seconds, to wait before executing the task
```